co-simulation tool/docker/ns3-docker/README.md

```diff
@@ -212,7 +217,13 @@ Please modify this file to:
 Once everything is ready, under directory `/ns-3-integration`, simply type:
 
 ```
-./mosaic.sh -s Co-simulation
+./mosaic.sh -s Town04_10
+./mosaic.sh -s Town04_200
+```
+
+After the run if there is no issue, commit docker container to docker image
+```
+docker container commit <container-id> ns-3-integration
 ```
 
-After the map in CARLA is loaded, you can start co-simulation with ns-3 in SUMO-GUI.
+Current CARMA XiL version not fully support NS-3 with CARMA-CARLA integration tool which is still under developing. User could run NS-3 co-simulation tool alone only for testing  by **`./mosaic.sh -s Co-simulation`**. 
```

Comment:
I'd recommend rephrasing a bit, does "Current CARMA XIL Version has a known issue with a port conflict between NS-3 and CARMA-CARLA integration see #39. The recommended workaround is to run exclusively one or the other module. To run with only NS-3module run **`./mosaic.sh -s Co-simulation`**."